db/schema.sql

```diff
@@ -11,15 +11,15 @@ DROP TABLE IF EXISTS payroll;
 
 -- create fresh tables
 CREATE TABLE "businesses" (
-  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
```

Comment:
I just changed the order of these so they look a little better. Not really related to the actual task, but I also didn't think it was worth making a whole separate task for. so here they are 😅